src/tools/ScenarioMeasurement/Startup/Crossgen2Parser.cs

```diff
@@ -60,60 +67,109 @@ public IEnumerable<Counter> Parse(string mergeTraceFile, string processName, ILi
     public sealed class EventParser
     {
         public string EventName { get; private set; }
+        public (int StartEventID, int StopEventID) EventID { get; private set; } // (StartEventID, StopEventID)
```

Comment:
I think this is an odd spot to use a tuple. I'd probably make this two separate fields.